### PR TITLE
current action now pulled from server and dead code removed

### PR DESCRIPTION
**Pull request by georgejdli**
It works
Not sure if CurrentAction should be refactored into it's own store and action file

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -26,3 +26,6 @@ build/Release
 # https://docs.npmjs.com/misc/faq#should-i-check-my-node-modules-folder-into-git
 node_modules
 dist/
+
+# OS generated files
+.DS_Store
```
Comment (austinpray) [inline .gitignore:31]: Generally this should be in a global .gitignore but I'll allow it :cop:

**Comment by austinpray** [comment]
Sweet, thanks!

File: .gitignore (hunk 1 above)
Comment (georgejdli) [inline .gitignore:31]: TIL about global .gitignore

Comment (austinpray) [inline .gitignore:31]: https://github.com/austinpray/dotfiles/blob/master/.gitignore_global

**Pull request merged**